### DISCUSSION TITLE
Charged Twice for Team Members

### DISCUSSION THREAD

**Discussion by odapplications** [vercel/community#2992] [Help] [answered]
### Page to Investigate

Teams

### Steps to Reproduce

I am being charged as the owner of a team only myself - $20 per month.
I invite someone to join the team costs an extra - $20 per month. 

This doesn't make sense since the owner isn't a team member they are orchestrating the team. And in some cases only 1 team member / dev is on the team. The owner isn't a developer and is just simply a project owner and not a developer. 

In this case, does the owner need to be on the team? Can the owner leave the team and still have full control over the members?

To charge $40 per month just to have 1 developer do rebuilds or check on status of the project is a bit over some. 

Why is the owner charged for being on the team, while already paying extra for a pro account? 

Its a bit confusing. A typical, setup with a third party developer normally would cost $10 per month for the pro account, and $20 for 1 team seat costing a total of $30 per month.  But in reality it costs $50 per month and is not clearly stated in the pricing that you are charging the owner of the team for a seat. So the expectation should not be $20 per month but $40 for 1 dev on the team.

**Comment by amyegan** [comment, marked as answer]
Hi @odapplications. I'm not sure I follow you with the monthly charges of $10 and $20 for Pro account and team seats. I'll answer based on what I think you're asking, and I apologize if I misunderstood. 

Pro teams are charged $20 per person each month. Owners and members are both charged at the same rate. There is no additional charge for the team itself, though individual features may have a cost associated with them. Please check the links below for more info:

- [Pricing information](https://vercel.com/pricing)
- [Team Members & Roles: Access Roles](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#)
- [Limits and feature pricing](https://vercel.com/docs/concepts/limits/overview#)

If this is a billing problem and you need help, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team. That will allow us to communicate privately to keep your personal and billing information secure.

**Reply by phantom3-com** [reply]
Hi @amyegan 

I am confused. 

I am on pro plan $20 /mo.  I asked the client to make a team and invite me. He needs a pro plan too which is $20/mo. 

Now we both in the team. Will he need to pay another $20/mo becuase he added me? bear in mind I have a pro plan

thanks

**Reply by SebasAran16** [reply]
I am having the same doubt. Have a Pro Plan paid of 20$  and when trying to invite someone to the team I am being warned:

"You are about to invite the following Team Members, are you sure you want to continue? $20 per user/month"

I understand that Pro Plan only includes one member of the team and will have to pay 20$ extra per each member added. Is this correct?

**Reply by amyegan** [reply]
That's essentially it. It's more like the team is being charged for its membership rather than an individual being charged for Pro feature access. The smallest "team" you could have is a single user, and a Pro team costs $20 per user each month. Which means the team would be charged $20/month for a single person team, $40/month for a two person team, and so on.

After the original post here, we've added some additional documentation about the Pro plans. You can find that info here:
- https://vercel.com/docs/accounts/plans/pro
- https://vercel.com/docs/accounts/plans/pro/billing

**Comment by odapplications** [comment]
It’s dumb the first user isn’t included and not clear that the first user
ie owner of the account is also being charged.


Worldwide Mobile Application Development & Marketing.

*P:* (714) 409-4719  *E:* ***@***.***   *W:* odapps.org

**Reply by amyegan** [reply]
Hey @odapplications. Can you elaborate? I'm not sure what you mean about the first user not being included.

There is no base cost for a team. The monthly bill only includes the number of people on the team and any add-on features you choose to include. So at $20/month, the first user is effectively included since the smallest possible team is just one person.

**Comment by odapplications** [comment]
Correct, you pay for premium to get 1 team member. But the owner being a
team member doesn’t make sense. They are the owner and should have access
but not considered a team member. The wording is misleading.

That’s like owning a team but you’re also a player on the team. That isn’t
normal in most cases.


Worldwide Mobile Application Development & Marketing.

*P:* (714) 409-4719  *E:* ***@***.***   *W:* odapps.org
